tests/Admin/Admin_Page_Tests.php

```diff
@@ -39,7 +39,8 @@ public function test_add_page() {
 		wp_set_current_user( $admin_user );
 		set_current_screen( 'dashboard' );
 
-		$page_hook    = $this->admin_page->add_page();
+		$this->admin_page->add_page();
+		$page_hook    = 'admin_page_plugin-check';
```

Comment:
See above. I don't know if or why PHPStan complains about it, but hard-coding this here is clearly worse than the code was before.

Reply:
This change was made for phpstan. I don't see the problem with hard coding this, why would it change?

Reply:
Hard-coding is a bad practice, since this value comes from WordPress core. It shouldn't change without a change of code, but by hard-coding the value here we make the test more error-prone. For example, if we change where the admin page is added in the class (e.g. to another admin menu), the original version of this test would just work. But with the value hard-coded here we would also need to update it, which is not optimal.

Reply:
Fixed.